games/nethack/Portfile

```diff
@@ -7,7 +7,8 @@ version            3.6.1
 categories         games
 platforms          darwin
 license            Copyleft
-maintainers        @Nax
+maintainers        @Nax \
+                   openmaintainer
```

Comment:
You wanted to add yourself as maintainer right? So you need to add your obfuscated email and Github Handle in brackets `{}`

`{@jflude <domain.com>:<name>}`

Reply:
Done

Reply:
Done